tests/chaos/src/cluster/bare/debug.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use datanode::datanode::DatanodeOptions;
+use frontend::frontend::FrontendOptions;
+use lazy_static::lazy_static;
+use meta_srv::metasrv::MetaSrvOptions;
+use regex::Regex;
+use servers::http::HttpOptions;
+
+use super::node::NodeInfo;
+use super::process::{list_processes_by_name, ProcessInfo};
+use crate::cluster::bare::node::Node;
+use crate::cluster::Cluster;
+use crate::error::Result;
+/// `DebugCluster` only returns existing node infos on the local machine.
+#[derive(Debug, Clone)]
+pub struct DebugCluster;
+
+const GREPTIME_DB_EXEC_FILENAME: &str = "greptime";
+
+lazy_static! {
+    static ref DATANODE_NODE_ID: Regex = Regex::new("^--node-id=([0-9]+)$").unwrap();
+}
+
+#[async_trait::async_trait]
+impl Cluster<Node> for DebugCluster {
+    async fn apply(&self) -> Result<()> {
+        Ok(())
+    }
+
+    async fn delete(&self) -> Result<()> {
+        Ok(())
+    }
+
+    async fn nodes(&self) -> Result<Vec<Node>> {
+        let processes = list_processes_by_name(GREPTIME_DB_EXEC_FILENAME.to_string())?;
+        // TODO(weny): add etcd nodes?
+        let mut nodes = Vec::with_capacity(processes.len());
+        for proc in processes {
+            let node = new_debug_node(proc.try_into()?);
+            nodes.push(node);
+        }
+
+        Ok(nodes)
+    }
+}
+
+fn parse_node_id(val: &str) -> u64 {
+    let node_id = DATANODE_NODE_ID.captures(val).unwrap()[1].to_string();
+    node_id.parse().unwrap()
+}
+
+fn new_debug_node(process: ProcessInfo) -> Node {
+    let info = match process.cmdline[1].as_str() {
+        "datanode" => {
+            // ["greptime", "datanode", "start", "--node-id=0", "--metasrv-addr=0.0.0.0:3002", "--rpc-addr=0.0.0.0:14100", "--http-addr=0.0.0.0:14300"]
+            let node_id = parse_node_id(&process.cmdline[3]);
```

Comment:
Well, we can't simply get node id like this. The `greptime` could start with just `-c=config.toml`. Can we pass these info from outside?